doc/README.md

```diff
@@ -0,0 +1,21 @@
+# Architecture Decision Records
+
+The directory [adr](adr) hold Architecture Decision Records that document major decisions made
+in the design of the NATS Server.
+
+A good intro to ADRs can be found in [Documenting Architecture Decisions](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions) by Michael Nygard.
```

Comment:
Interesting - the URL here doesn't load, and if I change to https I get a get back to error from chrome

Reply:
OK - after a super long wait, it loaded...